README.md

```diff
@@ -45,17 +45,27 @@ All members that have been inducted for standard allowance as per the [Fellowshi
 All members must serve a minimum period of service of 12 months at their current rank before seeking promotion to a higher rank. 
```

Comment:
```
  minPromotionPeriod: [
    0
    2,629,800
    2,629,800
    2,629,800
    2,629,800
    2,629,800
    2,629,800
    1,314,900
    2,103,840
  ]
```

Maybe I miss something, but how is `1 -> 2` not enforced? `0 -> 1` is correct.

Reply:
@bkchr @acatangiu 
Here is the info on Subsquare: https://collectives.subsquare.io/fellowship/core/params
Is Rank 1 = Index 0 and Rank 9 = Index 8? 🤔

Reply:
Ok, looks like UI problem then. Opened https://github.com/opensquare-network/subsquare/issues/4306 for it.

Reply:
> Ok, looks like UI problem then. Opened [opensquare-network/subsquare#4306](https://github.com/opensquare-network/subsquare/issues/4306) for it.

Thanks a lot! 🙏

Reply:
Fixed, we had a problem about calculating the min promotion period. We used the rank of a member instead of the `to_rank` to be promoted to. Thanks for all the info and discussions.

Reply:
> Fixed, we had a problem about calculating the min promotion period. We used the rank of a member instead of the `to_rank` to be promoted to. Thanks for all the info and discussions.

Thanks for the update.

I am still puzzled by the discrepancy in historical data for a member like Brenzi:
1) Was he inducted and promoted on **26th September 2023** as per this:  https://collectives.subsquare.io/fellowship/referenda/23 / https://collectives.polkassembly.io/referenda/23 ?

OR 

2) Was he inducted and promoted on **22nd February 2023** as per this: https://kusama.subsquare.io/fellowship/referenda/23?tab=timeline / https://kusama.polkassembly.io/member-referenda/23 ?

Please help. 🙏

Reply:
@anaelleltd Brenzi was added to fellowship collective on 26th September 2023. We can confirm this on [explorer](https://collectives.statescan.io/#/blocks/2194624?tab=events&page=1). Here he was only added to collectives pallet, not core pallet. We can check [here](https://collectives.subsquare.io/fellowship/core/feeds?who=12brenJreRTd6XsBMo3Ptu5VMUUycC53ipDxUy9Mvb4L1s1q) for his feeds in core pallet.

Reply:
> @anaelleltd Brenzi was added to fellowship collective on 26th September 2023. We can confirm this on [explorer](https://collectives.statescan.io/#/blocks/2194624?tab=events&page=1). Here he was only added to collectives pallet, not core pallet. We can check [here](https://collectives.subsquare.io/fellowship/core/feeds?who=12brenJreRTd6XsBMo3Ptu5VMUUycC53ipDxUy9Mvb4L1s1q) for his feeds in core pallet.

So, if he was imported to Fellowship Core at rank 1 less than 12 months ago, shouldn't his promotion period be ongoing, rather than showing complete (as on the screenshot)? I am just trying to work out if the [current Ref](https://collectives.subsquare.io/fellowship/referenda/139) to promote brenzi to rank 2 will execute without hiccups.

Thanks.

Reply:
![img_v3_02c6_cac43ad4-a468-422f-8aa7-2fbb9db4601g](https://github.com/polkadot-fellows/Evidences/assets/2264908/c9270f4a-1534-46b7-b708-72e39caf9ace)
@anaelleltd Upper image show brenzi's latest fellowship core status. When a member is imported to fellowship core, his/her `lastPromotion` is set 0. When `now` - `lastPromotion` >= `minPromotionPeriod`, the promotion will be available. So Brenzi is available for promotion.

Reply:
Some early joiners of the fellowship, were added/inducted straight to a certain level != 0, so they never went through a promotion so the on-chain counter is `0`.

Most other members, including future ones, were/will-be inducted as rank 0 and have to progress through the ranks using the complete process (including min period 1y between promotions).

Reply:
Thank you all.
I have added Polkadot-JS as a resource for checking member's status on the chain state.